src/js/popup.js

```diff
@@ -131,8 +131,7 @@ function send_error(message) {
     var origins = badger.getAllOriginsForTab(tabId);
     if(!origins){ return; }
     var version = chrome.runtime.getManifest().version;
-    //TODO "there's got to be a better way!"
-    var fqdn = tab.url.split("/",3)[2];
```

Comment:
For the future, this should be its own PR really (something like, "fix FQDN in error reports").